js/components/confirmation.jsx

```diff
@@ -8,6 +8,13 @@ import RequestSummary from './request_summary';
 
 function Confirmation({ agencyComponent, formData, requestForm, submissionResult }) {
   const { submission_id } = submissionResult;
+  const getDate = () => {
```

Comment:
It looks like in order to specify a format other than the default, you have to also pass in a language option (without that, at least, the `options` do nothing in my console). I don't want to format the date as `month/day/year` (the default) because that will be confusing outside of the US. :-/